ArcGISRuntimeSDKQt_CppSamples/Maps/MobileMap_SearchAndRoute/main.cpp

```diff
@@ -28,20 +28,21 @@
 
 int main(int argc, char *argv[])
 {
-    QGuiApplication app(argc, argv);
+  QGuiApplication app(argc, argv);
```

Comment:
spacing is off throughout this file. Please change it so the default indentation is 2 spaces